offchain/danaswap-api/test/TestUtil.purs

```diff
@@ -48,37 +72,47 @@ useRunnerSimple contract runner = do
 defaultWallet :: Array BigInt.BigInt
 defaultWallet = [ BigInt.fromInt 40_000_000, BigInt.fromInt 40_000_000 ]
 
-plutipConfig :: PlutipConfig
-plutipConfig =
-  { host: "127.0.0.1"
-  , port: UInt.fromInt 8084
-  , logLevel: Warn
-  -- Server configs are used to deploy the corresponding services.
-  , ogmiosConfig:
-      { port: UInt.fromInt 1338
-      , host: "127.0.0.1"
-      , secure: false
-      , path: Nothing
-      }
-  , ogmiosDatumCacheConfig:
-      { port: UInt.fromInt 10000
-      , host: "127.0.0.1"
-      , secure: false
-      , path: Nothing
-      }
-  , ctlServerConfig: Just
-      { port: UInt.fromInt 8083
-      , host: "127.0.0.1"
-      , secure: false
-      , path: Nothing
-      }
-  , postgresConfig:
+freePort :: Aff Int
+freePort = do
+  nextTry <- liftEffect $ randomInt 1024 0xFFFF
+  isGood <- isPortAvailable (UInt.fromInt nextTry)
+  if isGood then pure nextTry
+  else freePort
+
+plutipConfig :: Aff PlutipConfig
```

Comment:
lets call it `getRandomPortPlutipConfig` or something similar

Reply:
Is `getPlutipConfig` and a comment explaining what it does good? I don't like just saying random without explaining that it's just the ports and a name like `getPlutipConfigFreePorts` feels unwieldy.

Reply:
yes getPlutipConfig with a docstring is fine